src/System.ValueTuple/ref/System.ValueTuple.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <NuGetTargetMoniker Condition="'$(TargetGroup)' == 'netstandard1.0'">.NETStandard,Version=v1.0</NuGetTargetMoniker>
```

Comment:
This isn't needed it will get defaulted when TargetGroup is set to netstandard1.0.